Editor/Window/ManagedEC2Deployment.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Threading.Tasks;
+using AmazonGameLift.Editor;
+using UnityEngine;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Deployment
+    {
+        private readonly DeploymentSettings _model;
+        private readonly FleetParameters _parameters;
+        
+        internal ManagedEC2Deployment(DeploymentSettings model, FleetParameters parameters)
+        {
+            _model = model;
```

Comment:
please rename to `_deploymentSettings`. It's not apparent what model meanas without referring back to the type